src/renderer/transaction/swap/index.jsx

```diff
@@ -94,6 +94,8 @@ const styles = theme => ({
 	}
 });
 
+const KEY_ADDRESS = '0x4cc19356f2d37338b9802aa8e8fc58b0373296e7';
```

Comment:
This looks a bit ugly, if you don't want to deal with it now, maybe add a todo to think about it later at least

Reply:
Refactored.